src/components/AddTodoForm/AddTodoForm.tsx

```diff
@@ -0,0 +1,182 @@
+import { useCallback, useState } from 'react';
+import { User } from '../../types/User';
+import './index.scss';
+
+enum ChangeType {
+  TITLE,
+  USER,
+  COMPLETED,
+}
+
+type Props = {
+  handleSubmit: (
+    user: number,
+    title: string,
+    completed: boolean,
+  ) => void;
+  users: User[],
+};
+
+export const AddTodoForm = ({ handleSubmit, users }: Props) => {
+  const [title, setTitle] = useState('');
+  const [user, setUser] = useState(0);
+  const [completed, setCompleted] = useState(false);
+  const [errors, setErrors] = useState({
+    isUserNotSelected: false,
```

Comment:
not really good idea to start with negative in booleans, better to avoid it. It is a bit difficult to compile in my head, like `isNotSelected = false`. It means `isUserSelected = true`?

Reply:
But I'm trying to list down all possible errors. Then to maybe turn it into its own folder for errors, for reusability. Just to say what I had in mind

Reply:
🤔

Reply:
in this context you're right I guess.
Then I guess would be better to create separate validator for that and check all the fields on submit

Reply:
or validate fields separately after they're touched